assets/javascript/deadlineText.js

```diff
@@ -0,0 +1,44 @@
+//DO NOT CHANGE DELIMITERs
+const DELIMITER_BEG =
+    'DO NOT DELETE THIS TEXT AND INCLUDE A SPACE AFTER COLON: ';
+// 2 Ending because for some reason fetching from google doc has to
+const DELIMITER_END1 = '</span>';
+const DELIMITER_END2 = '"';
```

Comment:
Just to make things clear, the ending index will be at either `</span>` or at `"` on the Google Doc, whichever comes first?
Besides that, all is good.

Reply:
Yes because there is a weird thing that happens (i don't know why) but fetching the google doc has two potential results it can pull where one has after the text it ends with```</span>``` or the other with ```"```. That is why I added a couple of safety checks to make sure it doesn't mess up. I tried using google's own API to fetch the data making it more consistent and clean but it took 2 seconds to fetch so I opted to use this method instead